src/server/serviceProvider/transformers/Community/Reviews/searchReviews.transform.js

```diff
@@ -0,0 +1,88 @@
+import parseReview from '../../../parsers/review.parser';
+import workParser from '../../../parsers/work.parser';
+import {_} from 'lodash';
+
+const SearchReviewsTransform = {
+  event() {
+    return 'searchReviews';
+  },
+
+  requestTransform(event, {}, connection) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this pattern to not be empty. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3799)